packages/app/src/app/pages/Pro/Upgrade.tsx

```diff
@@ -83,16 +66,72 @@ export const ProUpgrade = () => {
     }
   }, [hasLoadedApp, location, setActiveTeam, personalWorkspaceId, dashboard]);
 
-  if (!hasLoadedApp || !isLoggedIn || !prices || !activeTeamInfo) return null;
+  const { isPersonalSpace, isTeamAdmin } = useWorkspaceAuthorization();
+  const { isFree, isPro } = useWorkspaceSubscription();
+  // const isFree = false; // DEBUG
+  // const isPro = true; // DEBUG
+
+  /**
+   * There is currently no way to know if teams have a custom subscription. This means we will
+   * always show the manage subscription button for the pro tier, together with the link to TypeForm
+   * to upgrade to a custom subscription. Keeping this variable here for future reference.
+   */
+  const hasCustomSubscription = false;
+
+  const checkout = useGetCheckoutURL({
+    team_id: isTeamAdmin && isFree ? activeTeam : undefined,
+    success_path: dashboardUrls.settings(activeTeam),
+    cancel_path: '/pro',
+    // recurring_interval: 'year', // TODO: defaulting to year does not enable the interval switch in stripe
```

Comment:
Waiting for an answer to see if we can enable the switch when passing a default interval. Right now I turned the default interval off to make sure the switch shows up.

Reply:
We're going with `month`, because this enabled the switch in Stripe to toggle between monthly and annual.